src/components/Navbar/index.tsx

```diff
@@ -60,19 +61,19 @@ export default function Navbar() {
   const menu = () => {
     return (
       <>
-        {menuItem('Projects', '/#/projects')}
-        {menuItem('FAQ', undefined, () => {
+        {menuItem(t`Projects`, '/#/projects')}
+        {menuItem(t`FAQ`, undefined, () => {
           window.location.hash = '/'
           setTimeout(() => {
             document
               .getElementById('faq')
               ?.scrollIntoView({ behavior: 'smooth' })
           }, 0)
         })}
-        {menuItem('Docs', 'https://docs.juicebox.money')}
-        {menuItem('Blog', 'https://blog.juicebox.money')}
+        {menuItem(t`Docs`, 'https://docs.juicebox.money')}
+        {menuItem(t`Blog`, 'https://blog.juicebox.money')}
         {menuItem('Discord', 'https://discord.gg/6jXrJSyDFf')}
-        {menuItem('Workspace', 'https://juicebox.notion.site')}
+        {menuItem(t`Workspace`, 'https://juicebox.notion.site')}
```

Comment:
Thoughts on removing this from the main nav?

Reply:
This was included a while back at the request of the folks organizing Notion, I think it helps with pointing newcomers there. Let's leave it for now, but throw out a question in the Discord if you're keen on removing it

Reply:
Thanks for context, no worries, I don't feel strongly!